cmd20.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+# Name: cmd20py
+# Authors: Luke Short | Joe R
```

Comment:
Separate authors into a dedicated `AUTHORS` file to avoid duplicating this in other files.

Reply:
Done as per PR: https://github.com/ekultails/cmd20py/pull/4